cms/static/js/views/assets.js

```diff
@@ -217,9 +233,13 @@ define(["jquery", "underscore", "gettext", "js/views/baseview", "js/models/asset
 
             startUpload: function (event) {
                 var file = event.target.value;
+
                 if (!this.largeFileErrorMsg) {
                     $('.upload-modal h1').text(gettext('Uploading'));
-                    $('.upload-modal .file-name').html(file.substring(file.lastIndexOf("\\") + 1));
+                    HtmlUtils.setHtml(
```

Comment:
Since file doesn't contain HTML, I'd just use this:

    $('.upload-modal .file-name').text(file.substring(file.lastIndexOf("\\") + 1));